antarest/core/config.py

```diff
@@ -155,6 +155,7 @@ class StorageConfig:
     auto_archive_dry_run: bool = False
     auto_archive_sleeping_time: int = 3600
     auto_archive_max_parallel: int = 5
+    snapshot_lifespan: int = 1
```

Comment:
lifespan sounds relevant, but I would prefer to use "retention" for consistency sake with the previous evolution on snapshots cleaning.
Also, because here we only have an integer, we don't know what's the unit, so we can postfix it with "days" too: `snapshot_retention_days`